snapcraft/plugins/catkin.py

```diff
@@ -38,6 +38,11 @@
       (list of strings)
       List of rosinstall files to merge while pulling. Paths are relative to
       the source.
+    - recursive-rosinstall:
```

Comment:
we need a consolidated strategy for these, wouldn't an enum be better long term to cater for all the flags or are boolean properties the way to go?

Asking for a friend ;-)